src/dodal/devices/aperturescatterguard.py

```diff
@@ -89,11 +88,13 @@ class ApertureScatterguard(InfoLoggingDevice):
     scatterguard = Cpt(Scatterguard, "-MO-SCAT-01:")
     aperture_positions: Optional[AperturePositions] = None
     TOLERANCE_STEPS = 3  # Number of MRES steps
+    ROBOT_LOAD_TOLERANCE = 5  # Number of mm from ROBOT_LOAD_Y
```

Comment:
Oh, sorry, I notice this before - we can't use a hardcoded position here because they need to be able to be adjusted in a configuration file (used by both GDA and Hyperion), so we load the positions in to `self.aperture_positions` on creating this device. 5 mm is also too big of a tolerance, I was suggesting 5 times the spatial resolution of the motor.

So then in `_get_current_aperture_position` you can compare `current_ap_y` to that dynamically loaded position value with the calculated tolerance loaded from the motor, something like:

```Python
def _get_current_aperture_position(self) -> SingleAperturePosition:
    ...
    robot_load_ap_y = self.aperture_positions.ROBOT_LOAD.location.aperture_y
    tolerance = self.TOLERANCE_STEPS * self.aperture.y.motor_resolution.get()
    ...
    elif current_ap_y <=  + robot_load_ap_y + tolerance:
    ...
```

Reply:
Thanks @dperl-dls - I've added in these abstractions now and just updated the tests to also call them (instead of the hard coded within ant outside tolerance values)